### PR TITLE
docs: Update Placeholder Image and Lorem Ipsum shortcode pages

### PR DESCRIPTION
**Pull request by mcanouil**
This pull request adds a highlight for the two new shorcodes: "placeholder" and "lipsum".

### CODE REVIEW

**Comment by cderv** [comment]
Same as the other PR, I would say we should add this to prerelease website now that we have it (on prerelease branch). 

I let @cwickham tell us if we need to apply that now. 

Thanks for documenting the feature !

**Comment by cwickham** [comment]
Hmmm...we actually already have some docs on pre-release: https://github.com/quarto-dev/quarto-web/commit/f4e88bac425c983102fbab87b1d5f3c9ac780d60

I would like to see:
* This PR target the `prerelease` branch
* Any extra context/examples in this PR merged into the files `docs/authoring/lipsum.qmd` and `docs/authoring/placeholder.qmd`
* `_highlights.yml`  link directly to `docs/authoring/lipsum.qmd` and `docs/authoring/placeholder.qmd`.

As an aside, based on discussion in the sync last week, going forward the pre-release highlights page will live on the `prerelease` branch. For 1.5, perhaps we should edit the pre-release highlights page on `main` to just point to the pre-release docs highlight page.

**Comment by cderv** [comment]
> For 1.5, perhaps we should edit the pre-release highlights page on main to just point to the pre-release docs highlight page.

Good idea ! 👍

**Comment by mcanouil** [comment]
I came back to this and refactored to combine with the two pages that were created since my original PR was made.

**Comment by mcanouil** [comment]
I added (replacing the alias to "extension") a shortcode page in "authoring" as it's more likely users will want to find those in authoring rather than going on the "create shortcode" page.

**Comment by mcanouil** [comment]
I missed the sidebar update...

**Comment by mcanouil** [comment]
By the way, the sidebar guide and the custom listing does not match:

| Sidebar | Listing |
|--------|--------|
| image | image |

**Comment by cderv** [comment]
> By the way, the sidebar guide and the custom listing does not match:

Probably true for other categories too

**Comment by mcanouil** [comment]
> > By the way, the sidebar guide and the custom listing does not match:
> 
> Probably true for other categories too

We probably should fix that in another PR.

**Comment by cwickham** [comment]
> > > By the way, the sidebar guide and the custom listing does not match:
> > 
> > 
> > Probably true for other categories too
> 
> We probably should fix that in another PR.

This is by design. For the Guide Page, we only list some options for each section. Open for discussion but let's make it a separate issue.

> I added (replacing the alias to "extension") a shortcode page in "authoring" as it's more likely users will want to find those in authoring rather than going on the "create shortcode" page under "extension".

This is an excellent idea! But, perhaps we should list it earlier since the pages Videos and Embeds, come earlier in this Authoring section and also use short codes. (Technically, we first mention them on Markdown Basics, but it is just the `kbd` one 🤷🏻‍♀️ ). How about putting it after Diagrams?

I'd also like to see `lipsum.qmd` and `placeholder.qmd` show up somewhere in the sidebar nav. Perhaps we can nest them under Shortcodes. I don't want to nest Videos and Embeds.

File: docs/authoring/lipsum.qmd [hunk 1]
```diff
@@ -6,7 +6,8 @@ aliases:
 
 ## Overview
 
-You can include placeholder text ("lorem ipsum") in your Quarto documents by using the `lipsum` shortcode.
+When you need to visualize how text will look on your website, the `{{{< lipsum >}}}` shortcode comes in handy.
```
Comment (cwickham) [inline docs/authoring/lipsum.qmd:9]:
```suggestion
When you need to visualize how text will look in your document or your website, the `{{{< lipsum >}}}` shortcode comes in handy.
```

File: docs/authoring/placeholder.qmd [hunk 2]
```diff
@@ -6,7 +6,8 @@ aliases:
 
 ## Overview
 
-You can include placeholder images in your Quarto documents by using the `placeholder` shortcode.
+Th `{{{< placeholder >}}}` shortcode generates a placeholder image, which is incredibly useful when you're designing your website layout but the final images aren't ready yet.
```
Comment (cwickham) [inline docs/authoring/placeholder.qmd:9]:
```suggestion
The `{{{< placeholder >}}}` shortcode generates a placeholder image, which is incredibly useful when you're designing your document or website layout but the final images aren't ready yet.
```

File: docs/authoring/lipsum.qmd [hunk 3]
```diff
@@ -16,7 +17,16 @@ Here's an example of `lipsum` generating two paragraphs of text, by adding `{{{<
...
+::: {layout-ncol=1}
+```markdown
+{{{< lipsum 1 >}}}
+```
+:::: {.border}
+{{< lipsum 1 >}}
+::::
+:::
+
 `lipsum` takes a single parameter, in two possible formats:
```
Comment (cwickham) [inline docs/authoring/lipsum.qmd:29]: Pretty sure this parameter is optional:
```suggestion
`lipsum` takes an optional parameter, in two possible formats:
```

File: docs/authoring/placeholder.qmd [hunk 4]
```diff
@@ -21,6 +22,16 @@ You can customize the size and format of the image by providing parameters to th
 ## Usage
 
 The `placeholder` shortcode takes a variable number of parameters, ranging from zero to two.
```
Comment (cwickham) [inline docs/authoring/placeholder.qmd:24]: It feels like the example shows three parameters? Oh, I see, there is a distinction made between parameters and keywords.  Perhaps easier to say:
```suggestion
Additional arguments control the size and format of the placeholder image:
```

Comment (mcanouil) [reply docs/authoring/placeholder.qmd]: Would the following work for you?
0a44b7cf872dbb69034bf120c8569595e2e4f2ba

**Comment by mcanouil** [comment]
@cwickham Don't hesitate to take over the PR in order to make small rewording as needed.

**Comment by cwickham** [comment]
> `docs/authoring/markdown-basics.qmd` contains also videos after diagram and the kbd shortcode at the end.

Oh yeah, it does. I will add a few links from the word shortcode in `docs/authoring/markdown-basics.qmd` to the new Authoring > Shortcodes page. 

Then I'll merge. Thanks @mcanouil !

**Pull request merged**